1/README.md

```diff
@@ -2,6 +2,12 @@
 
 This image contains an installation Nginx 1.x.
 
+> WARNING: This Dockerfile installs NGINX Prometheus Exporter and should be used
```

Comment:
This must be generated from README.yaml. I think you can simple drop it, since Dockerfile contains this note already.